book/src/bindings/r.md

```diff
@@ -3,9 +3,17 @@
 R bindings for [`prql-compiler`](https://github.com/prql/prql/).
 Check out <https://eitsupi.github.io/prqlr/> for more context.
 
+```admonish note
+`prqlr` is generously maintained by [@eitsupi](https://github.com/eitsupi) in the
+[eitsupi/prqlr](https://github.com/eitsupi/prqlr) repo. Please feel free to raise any
+questions or issues in the repo.
```

Comment:
I want this to sound like "@eitsupi is great, check out their work" not like "don't bring your complaints to me"! 

Let me know if this strikes the right tone!